publish/prepare-artifacts.proj

```diff
@@ -1,6 +1,24 @@
 <Project>
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
+	<UsingTask TaskName="GenerateChecksums" AssemblyFile="$(LocalBuildToolsTaskFile)" />
```

Comment:
A few tabs snuck in.

Reply:
Ugh, copy paste from web browser will do that 😆